pypeit/traceslits.py

```diff
@@ -152,13 +152,17 @@ def __init__(self, spectrograph, par, binning=None, det=1, master_key=None, mast
     @property
     def nslit(self):
         """
-
         Returns:
             int: Number of slits currently identified
-
         """
         return 0 if self.slit_left is None else self.slit_left.shape[1]
 
+    def _edgearr_single_slit(self):
```

Comment:
traceslits is deprecated.  If you want this functionality,
you'll need to add it to `edgetrace`